### DISCUSSION TITLE
Truenas Scale App - Failure while importing uploaded files

### DISCUSSION THREAD

**Discussion by ctag** [photoprism/photoprism#2354] [General]
Hi,

I'm using Photoprism from the official Truenas apps list. Has been great so far, but throws an error every time I try uploading images to an album, though the images do appear after some time. 

I'm curious if anyone else is seeing this, as searching the error message hasn't brought anything up.

![2022-05-24_08-26](https://user-images.githubusercontent.com/6200085/170047346-c52fa2c8-dd23-4ab5-8ca4-e5408006714c.png)

**Comment by graciousgrey** [comment]
Do you see related errors on the error page or logs under logs ?

**Comment by ctag** [comment]
Thank you for the suggestion.

Under Library I see an Errors page, but not Logs. Logs is already enabled in settings. I'll try restarting the container and continuing to upload.

Only one error is listed: 
`import: /photoprism/import/upload/1653397968722 does not exist `

![2022-05-24_09-17](https://user-images.githubusercontent.com/6200085/170058562-be798cc6-d00e-47c1-b359-2b0c7406b5e8.png)

Edit: When uploading fewer (< ~30?) images, the import appears to succeed without error notifications.

**Reply by graciousgrey** [reply]
We don't have a upload limit build in. I also cannot reproduce this on our public demo. I can upload way more than 30 images without error.

There might be several reasons for this related with your setup (proxy settings, permission issues, scripts running in the background....) 

Alternatively you can add your files to the import folder using webDAV (https://docs.photoprism.app/user-guide/sync/webdav/), this is much faster than the  web upload. You would need to add the files to albums manually after import.

**Reply by ctag** [reply]
Thank you for taking a look at this. 

I'm just reporting back that using the import folder (manually copying into) worked well and I haven't seen that error again.
